cluster-scope/base/core/persistentvolumeclaims/image-registry-storage/persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
```

Comment:
does this claim need to be in namespace `openshift-image-registry`?

Reply:
Yes!